File: abrpoint.client/src/models/PosteDto.ts
export interface PosteDto{
    codposte: string,
    soccod: string,
    libposte: string,
    avantent: number,
    apresent: number,
    avantsort: number,
    apressort: number,
    jourhdmat: string,
    jourhfmat: string,
    jourhdam: string,
    jourhfam: string,
    jourrepos: string,
    jourrepas: number,
    jourhdrep: string,
    jourhfrep: string,
    jourhdematin: string,
    jourhfematin: string,
    jourhdeamidi: string,
    jourhfeamidi: string,
    arrondi: number,
    maxhrejour: string,
    minhjour: number,
    minhdemijour: number,
    jourdouche: number
}
export interface UpdatePoste{
    apresent:number
    apressort:number
    avabon:number
    avamn:number
    avantent:number
    avantsort:number
    soccod:string
    codposte:string
    libposte:string
    reposApres:string
    reposAvant:string
    retmin:number
    retminam:number
    retsanc:number
    retsancam:number
}

File: ABRPOINT.Server/Dtaos/PosteDto.cs
using System.Text.Json.Serialization;

namespace ABRPOINT.Server.Dtaos;
public class PosteDto
{
    [JsonPropertyName("catcod")]
    public string? Catcod { get; set; }
    public string? Codposte { get; set; }
    public string? Soccod { get; set; }
    public string? Libposte { get; set; }
    public int? Avantent { get; set; }
    public int? Apresent { get; set; }
    public int? Avantsort { get; set; }
    public int? Apressort { get; set; }
    public string? Jourhdmat { get; set; }
    public string? Jourhfmat { get; set; }
    public string? Jourhdam { get; set; }
    public string? Jourhfam { get; set; }
    public string? Jourrepos { get; set; }
    public int? Jourrepas { get; set; }
    public string? Jourhdrep { get; set; }
    public string? Jourhfrep { get; set; }
    public string? Jourhdematin { get; set; }
    public string? Jourhfematin { get; set; }
    public string? Jourhdeamidi { get; set; }
    public string? Jourhfeamidi { get; set; }
    public int? Arrondi { get; set; }
    public string? Maxhrejour { get; set; }
    public float? Minhjour { get; set; }
    public float? Minhdemijour { get; set; }
    public float? Jourdouche { get; set; }

}